gamedata/scripts/magazines.script

```diff
@@ -52,6 +52,35 @@ end
 --------------------------------------
 -- SECTION mag un/loading functions --
 --------------------------------------
+
+
+-- Get the time delay between loading/unloading rounds from/to magazine
+--
+-- **mag_section** *(string)* = section of the mag to get **load_delay** param if **ignore_loaddelay** is false 
+--
+-- **is_unload** *(bool)* = should we get mag unload time
+function get_mag_time(mag_section,is_unload)
+    is_unload=is_unload or false
+
+	local load_factor=magazines_mcm.get_config('mag_loadtime_factor')
+	local unload_factor=magazines_mcm.get_config('mag_unloadtime_factor')
+	local default_delay=0.4
```

Comment:
try:
`default_delay=SYS_GetParam(2, "tch_mag_base", "load_delay") or 0.4`

the magazine base delay is the actual default the hard coded one should only protect against someone (ok me) making a typo.